app/src/main/res/values/strings.xml

```diff
@@ -6,10 +6,18 @@
     <string name="common_delete">Delete</string>
     <string name="common_remove">Remove</string>
     <string name="common_next">Next</string>
+    <string name="common_submit">Submit</string>
 
-    <string name="contestant_welcome_message">Welcome to\n%1$s!</string>
-    <string name="name_entry">Name your submission</string>
     <string name="join_contest">Join Contest</string>
     <string name="create_contest">Create Contest</string>
     <string name="create_join_prompt">Create and join contests</string>
+    <string name="contestant_welcome_message">Welcome to\n%1$s!</string>
+    <string name="name_entry">Name your submission</string>
+    <string name="entry_image_question">Add an image for Dank Chili?</string>
```

Comment:
Yes!!! I definitely need some kind of check for this.
Do you think it's appropriate to break this View method in two, and test each one?
```

    public void showEntryName(String entryName) {
        previewLabelTextView.setText(getResources().getString(R.string.entry_image_preview_caption, entryName));
        questionTextView.setText(getResources().getString(R.string.entry_image_question, entryName));
    }
```

Or can I leave that to a View test?